### PR TITLE
WIP: Allow import of NAWS export in installation wizard

### PR DESCRIPTION
**Pull request by jbraswell**

### CODE REVIEW

File: main_server/local_server/install_wizard/installer.js [hunk 1]
```diff
@@ -737,10 +747,10 @@ function gmScriptLoadCompletion()
 *   \returns a new XMLHTTPRequest object.                                                   *
 ********************************************************************************************/
     
-function BMLT_Installer_AjaxRequest(
+function BMLT_Installer_AjaxPostRequest(
```
Comment (jbraswell) [inline main_server/local_server/install_wizard/installer.js]: All usages of this function were for `POST`, so I simplified it by removing the `method` option.

**Comment by pjaudiomv** [comment]
None of your commits are verified, Please sign your commits and re open.

File: main_server/local_server/install_wizard/installer_ajax.php [hunk 2]
```diff
@@ -30,6 +30,11 @@
 require_once(dirname(__FILE__).'/../../server/shared/classes/comdef_utilityclasses.inc.php');
 require_once(dirname(__FILE__).'/../../server/shared/Array2Json.php');
 
+function generateRandomString($length = 10)
```
Comment (jbraswell) [inline main_server/local_server/install_wizard/installer_ajax.php]: Used to generate random passwords. This is not really a cryptographically secure random string generator, but it's close enough for BMLT purposes.

**Comment by jbraswell** [comment]
- [ ] Write meeting creation code
- [ ] Remove all TODOs
- [ ] Front-end logic for handling situation where config couldn't be written
- [ ] Front-end logic for displaying import errors
- [ ] Figure out what to do if an import fails - should it delete the config file and de-initialize the db? Should it just delete the imported data?

**Pull request merged**